qiskit/assembler/disassemble.py

```diff
@@ -50,7 +50,7 @@ def _experiments_to_circuits(qobj):
             conditional = {}
             for i in x.instructions:
                 name = i.name
-                if i.name == 'id':
+                if i.name == 'i':
                     name = 'iden'
```

Comment:
Should this case be deleted? The name should stay `'i'`, or why was it previously renamed to `'iden'`?

Reply:
Good catch. Yes, looks like this should be deleted since now i.name is the same as the circuit attribute.